sagemaker-lineage/sagemaker-lineage-cross-account-with-ram.ipynb

```diff
@@ -117,13 +117,11 @@
     "\n",
     "consumer_account_number = \"<AWS_ACCOUNT>\"\n",
     "\n",
-    "consumer_account = f\"arn:aws:iam::{consumer_account_number}:root\"\n",
-    "\n",
     "# Create resource share\n",
     "response = ram.create_resource_share(\n",
     "    name=\"sm_lineage_sharing\",\n",
     "    resourceArns=[lineage_group_arn],\n",
-    "    principals=[consumer_account],\n",
+    "    principals=[consumer_account_number],\n",
```

Comment:
I see that the phrase account "number" is used ~3 times in the notebook - more standard to say account ID / account_id?

Reply:
ya `account id` looks more correct